js/scripts.js

```diff
@@ -33,19 +33,21 @@ var popupOpen = false;
 // Portraits settings
 var portraitsBucketName = 'seniormap-portraits';
 
-// Sheets settings
-var institutionDataSheet = '1qEcBuuRtQT-hE_JyX6SlMxTodvXCtAXX1LSB4ABBlXU';
-var yearDocumentsSheet = '1VZmrdC-rm6noqxMoFWiPimOiM3-zmhk5kOmJ8RppU9w';
+// All google sheet published urls. These can be found in the shared seniormap google drive.
+var institutionDataSheetCoordinates = '2PACX-1vQnBlXwk6cpRpt-IVttNNCBvgySJl_JDVursNLtlB7IWtgTU5fBgldmFGudv-6oEwe-LeP8T-mzqylx/pub?gid=0';
+var institutionDataSheetLogos = '2PACX-1vQnBlXwk6cpRpt-IVttNNCBvgySJl_JDVursNLtlB7IWtgTU5fBgldmFGudv-6oEwe-LeP8T-mzqylx/pub?gid=933441586';
+var yearDocumentsSheetMain = '2PACX-1vRIX7bc0mlzPt-j5OWAIOYZsUEsrK5XWM_Q7hH4Zy-XSXkeP55CLXJv6vSRtZLZZ19v_OAA4yqzGCDU/pub?gid=0';
+var yearDocumentsSheetStats = '2PACX-1vRIX7bc0mlzPt-j5OWAIOYZsUEsrK5XWM_Q7hH4Zy-XSXkeP55CLXJv6vSRtZLZZ19v_OAA4yqzGCDU/pub?gid=1838156495';
 // README FLAG: A new year and datasheet URL must be added to this array for the script to access the data on the spreadsheet
 var yearDocuments = new Map([
     ['2019', {
```

Comment:
If we do convert to using integers instead of strings, we'll have to change these.